src/content/concepts/why-webpack

```diff
@@ -0,0 +1,55 @@
+---
+title: Why Webpack
+sort: 7
+contributors:
+- debs_obrien
+---
+
+##Why webpack?
+
+To understand why you should use webpack you first have to understand the origins of how we use JavaScript on the web. 
+
+JavaScript is just a script and there are 2 ways to use it in the browser, loaded as a script tag or written directly inside your html. The problems that this causes is that it doesn't scale. You either have too many scripts that you try to load from your script tags in the html and this causes a huge performance problem. There is only a certain amount of concurrent requests that can fetch data at a single time so breaking your application into a hundred JavaScript files and loading them in the browser is a huge performance problem. One big JS file results in unmaintainable scripts that causes problems in scope, size, readability, fragility and monolith files.
+
+
+##IIFE's - Immediately invoked function expressions
+
+IIFE's solve big problems with regards to scoping. Every file is treated as an IIFE which allows us to concatenate or safely combine files without concern of scope collision. Wow, we can now write 100 files and squish them together without concerns that we are going to knock out a bunch of variables and scopes.
+
+This lead to tools like Make, Gulp, Grunt, Broccoli, Brunch and StealJS being created. The purpose of these tools/task runners is to concatenate the files together in order to solve these problems. However there are still problems. With these tools a full rebuild is needed every time. Anytime you want to change one file you have to rebuild the whole thing. This also leads to a lot of dead code. Concat doesn't help the usages across files. If you are concatenating files together how do you remove code your not using? How do you even know what code your not using?
```

Comment:
`Concat doesn't help the usages across files`, it would be good to rephrase this oration, maybe changing Concat to Concatenate.

Also, please remember this documentation is used by thousands non native, non advanced English speakers, it really helps use simple words.

Reply:
`How do you even know what code your not using?` -> `How do you even know what code you are not using?`

Reply:
done